expression/builtin_info.go

```diff
@@ -589,3 +596,63 @@ func (b *builtinRowCountSig) evalInt(_ chunk.Row) (res int64, isNull bool, err e
 	res = int64(b.ctx.GetSessionVars().StmtCtx.PrevAffectedRows)
 	return res, false, nil
 }
+
+type tidbDecodeKeyFunctionClass struct {
+	baseFunctionClass
+}
+
+func (c *tidbDecodeKeyFunctionClass) getFunction(ctx sessionctx.Context, args []Expression) (builtinFunc, error) {
+	if err := c.verifyArgs(args); err != nil {
+		return nil, err
+	}
+	bf := newBaseBuiltinFuncWithTp(ctx, args, types.ETString, types.ETString)
+	sig := &builtinTiDBDecodeKeySig{bf}
+	return sig, nil
+}
+
+type builtinTiDBDecodeKeySig struct {
+	baseBuiltinFunc
+}
+
+func (b *builtinTiDBDecodeKeySig) Clone() builtinFunc {
+	newSig := &builtinTiDBDecodeKeySig{}
+	newSig.cloneFrom(&b.baseBuiltinFunc)
+	return newSig
+}
+
+// evalInt evals a builtinTiDBIsDDLOwnerSig.
+func (b *builtinTiDBDecodeKeySig) evalString(row chunk.Row) (string, bool, error) {
+	s, isNull, err := b.args[0].EvalString(b.ctx, row)
+	if isNull || err != nil {
+		return "", isNull, err
+	}
+	return decodeKey(b.ctx, s), false, nil
+}
+
+func decodeKey(ctx sessionctx.Context, s string) string {
+	key, err := hex.DecodeString(s)
+	if err != nil {
+		ctx.GetSessionVars().StmtCtx.AppendWarning(errors.New("the key is a invalid record/index key"))
+		return s
+	}
+	// Auto decode byte if needs.
+	_, bs, err := codec.DecodeBytes([]byte(key), nil)
+	if err == nil {
+		key = bs
+	}
+	// Try decode as table record key.
+	tableID, handle, err := tablecodec.DecodeRecordKey(key)
+	if err == nil {
+		return "tableID=" + strconv.FormatInt(tableID, 10) + ", handle=" + strconv.FormatInt(handle, 10)
```

Comment:
how about `s/handle/_tidb_rowid/`?

Reply:
If the primary key is the handle, there is no `tidb_rowid`  column.
So i think `handle` is better.

Reply:
From the user's perspective, we should provide a `_tidb_rowid` column if the PK is handle.

Reply:
OK, Done.